client/src/components/Draftail/sources/ModalWorkflowSource.test.js

```diff
@@ -3,6 +3,7 @@ import { shallow } from 'enzyme';
 
 import ModalWorkflowSource, { getChooserConfig, filterEntityData } from './ModalWorkflowSource';
 import * as DraftUtils from '../DraftUtils';
+import { DraftUtils as DraftailUtils } from 'draftail';
```

Comment:
hahahaha, looking at this here it’s so obvious I should’ve called this API DraftailUtils to start with.